tests/tests_fabric/strategies/test_fsdp_integration.py

```diff
@@ -621,8 +621,7 @@ def test_clip_gradients(clip_type, precision):
     optimizer.zero_grad()
 
 
-# TODO: Support checkpoint consolidation with PyTorch >= 2.2
-@RunIf(min_cuda_gpus=2, standalone=True, min_torch="2.1.0", max_torch="2.2.0")
+@RunIf(min_cuda_gpus=2, standalone=True, min_torch="2.3.0")
```

Comment:
I checked that the test passes locally. It won't run in our CI until we add testing with nightly, but that's ok for now.